source/sync/data-model/update-data-model/update-schema-production-apps.txt

```diff
@@ -1,20 +1,430 @@
+           banner, click :guilabel:`Review Draft & Deploy`.
+
+      4. If you *only* made non-breaking changes, your changes are automatically
+         applied to the current schema version without any additional handling.
+         A new schema version is *not* created.
+
+         .. note:: Added Elements are Not Available to Previous Versions
+
+            Because additive changes are applied to the current schema, clients
+            must use a schema version equal to or greater than the
+            current version to reference any elements added as part of
+            your changeset.
+
+      5. If you made at least one schema versioning or breaking change, a
+         confirmation window displays with a summary of your changes.
+
+   .. step:: Confirm and Deploy Changes
+
+      From the confirmation window, review the :guilabel:`Schema Version` and
+      :guilabel:`Change Summary` information:
+
+      - The current schema version, and the incremented number of the
+        new schema created upon deployment (if applicable).
+      - The type of change, labeled as ADDITIVE, SUPPORTED, or UNSUPPORTED.
+      - A description of the changes to the noted object type or field.
+      - Any notes or suggested actions to take for a given change. For example,
+        if App Services is expecting a default value based on the change you
+        made, the note suggests that you define a default value.
+
+      How you deploy your changes depends on the changes in your changeset:
+
+      .. tabs::
+
+         .. tab:: Schema Versioning
+            :tabid: supported
+
+            If your changeset includes at least one schema versioning change,
+            then all changes (including any non-breaking changes) are applied
+            to a new compatible schema version.
+
+            1. Review your changes, which may include non-breaking changes. We
+               recommend that you resolve any suggested actions noted before you
+               deploy your changes.
+
+            2. Click :guilabel:`Deploy`.
+
+            3. Your changes are applied to the new schema version. The new
+               version is compatible with previous versions.
+
+         .. tab:: Breaking
+            :tabid: unsupported
+
+            If your changeset includes at least one breaking change, then all
+            changes (including any non-breaking and schema versioning changes)
+            are applied in a single changeset. Upon deployment of breaking
+            changes, App Services terminates and re-initializes Device Sync.
+
+            1. Review your changes, which may include non-breaking and schema
+               versioning changes. We recommend that you resolve any suggested
+               actions noted before you deploy your changes.
+
+               Breaking changes may or may not increment the schema version:
+
+               - If the current schema version is greater than 0, App Services
```

Comment:
I don't understand the 0 stuff - maybe we should talk this through? If the current schema version is 0, but you can make breaking changes and that doesn't increment the schema - how does a schema version ever move from 0 to 1?

Reply:
with a schema versioning change. making a breaking change to a v0 schema only breaks the v0 schema rather than creating a new incompatible v1. (confirmed that this only applies for v0)